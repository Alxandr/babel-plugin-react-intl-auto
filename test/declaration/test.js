import test from 'ava'
import macro from '../util'

test('declaration', macro, __dirname)
