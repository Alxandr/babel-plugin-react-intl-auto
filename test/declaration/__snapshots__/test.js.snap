// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`declaration 1`] = `
"'use strict';

var _reactIntl = require('react-intl');

const messages = {
  hello: {
    'id': 'test.declaration.hello',
    'defaultMessage': 'hello'
  },
  world: {
    'id': 'test.declaration.world',
    'defaultMessage': 'hello world'
  }
};

(0, _reactIntl.defineMessages)(messages);"
`;
